### ISSUE TITLE
3rd Community Review of Genesis

### ISSUE THREAD

**Issue by Chuangshi1** [filecoin-project/Allocator-Governance#237]
Allocator Application：https://github.com/filecoin-project/notary-governance/issues/1019
First Community Diligence: #64 
Second Community Diligence: #154 
This round new current client: https://github.com/Chuangshi1/Genesis/issues/44
3.5PiB has been  allocated to the client.
Please check it. Hope go to the next round.

**Comment by filecoin-watchdog** [comment]
@Chuangshi1,
[Allocator Application](https://github.com/filecoin-project/notary-governance/issues/1019)
[Compliance Report](https://compliance.allocator.tech/report/f03019845/1733791903/report.md)  
[1st review](https://github.com/filecoin-project/Allocator-Governance/issues/64)
[2nd Review](https://github.com/filecoin-project/Allocator-Governance/issues/154)  
1st Review score: 5PiB
2nd Review score: 5PiB

5 PiB granted to one new client:
| Client Name | DC | status |
| -------- | ------- | ------- |
| JANetwork | 5PiB | New |

[JANetwork](https://github.com/Chuangshi1/Genesis/issues/44)
* The client intended to store 4 replicas of 200 TiB, totaling 0.8 PiB of data, not the requested 5 PiB.
* Information regarding data preparation is either missing or insufficient.
* The client had previously requested DataCap for the same dataset:[ Issue # 1679](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1679). This was not disclosed in the application form. It only came to light during the KYC process (the client wanted to reuse the KYC from LDN).
* KYC was conducted manually due to the client’s lack of familiarity with Gitcoin Passport.
* The Storage Providers used for deals align with the provided list.
* Good retrieval rates and data replication.
* In the allocator application, the client committed to require 5+ SPs and 5+ replicas. However, this application only includes 4 SPs and 4 replicas. 
The client requested 5 PiB of DataCap and has already received this amount. However, not all allocated DataCap has been sealed yet, leaving room for adjustments, which the allocator should actively monitor.

---
Another thing is, please refer to this comment https://github.com/filecoin-project/Allocator-Governance/issues/154#issuecomment-2485893903 
The gov team should check up on this as well.

**Comment by Chuangshi1** [comment]
> @Chuangshi1, [Allocator Application](https://github.com/filecoin-project/notary-governance/issues/1019) [Compliance Report](https://compliance.allocator.tech/report/f03019845/1733791903/report.md) [1st review](https://github.com/filecoin-project/Allocator-Governance/issues/64) [2nd Review](https://github.com/filecoin-project/Allocator-Governance/issues/154) 1st Review score: 5PiB 2nd Review score: 5PiB
> 
> 5 PiB granted to one new client:
> 
> Client Name	DC	status
> JANetwork	5PiB	New
> [JANetwork](https://github.com/Chuangshi1/Genesis/issues/44)
> 
> * The client intended to store 4 replicas of 200 TiB, totaling 0.8 PiB of data, not the requested 5 PiB.
> * Information regarding data preparation is either missing or insufficient.
> * The client had previously requested DataCap for the same dataset:[ Issue # 1679](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1679). This was not disclosed in the application form. It only came to light during the KYC process (the client wanted to reuse the KYC from LDN).
> * KYC was conducted manually due to the client’s lack of familiarity with Gitcoin Passport.
> * The Storage Providers used for deals align with the provided list.
> * Good retrieval rates and data replication.
> * In the allocator application, the client committed to require 5+ SPs and 5+ replicas. However, this application only includes 4 SPs and 4 replicas.
>   The client requested 5 PiB of DataCap and has already received this amount. However, not all allocated DataCap has been sealed yet, leaving room for adjustments, which the allocator should actively monitor.
> 
> Another thing is, please refer to this comment [#154 (comment)](https://github.com/filecoin-project/Allocator-Governance/issues/154#issuecomment-2485893903) The gov team should check up on this as well.

Thank you so much that tell us the problem. Here are the explanation from us that help us understand each other better:
1.  "totaling 0.8 PiB of data, not the requested 5 PiB" I don't exactly know how to calculate the total DC. If it means such as 500TiB*10 replicas=5PiB.  
![image](https://github.com/user-attachments/assets/e1e9971d-7ca5-4336-8be4-18789b68efd9)
And the old LDN that they applied was supported before, maybe i am mistaked here. But thank u for pointing it, we will attention in the future.
2.About the KYC, they tell us it has been KYC before and passed by the Kevzak. So we accept the way that they send the file to our slack to check them. If you need to check them, i can send them to you. And i think it is a heartwarming way for client that don't know how to KYC. 
![image](https://github.com/user-attachments/assets/3f526903-ace0-4235-86b3-8d143699ca91)
![image](https://github.com/user-attachments/assets/af82888a-d003-4e26-8335-eb2b848292e5)
3.The client proactively disclosed the past LDN (1679)。But it was stoped because the V5 new Allocator pattern, so i pay more attention to the SP information and CID report.
![image](https://github.com/user-attachments/assets/4af36099-1bf7-4ad8-b212-50f2bb8ce268)
4.Client would like to find more different regions SP and add here. 
![image](https://github.com/user-attachments/assets/3ea18a5a-f63c-407d-96d3-6db2d4c8ee40)
What we do next: 
1 Strengthen information disclosure: Advise customers to ensure that all relevant information is disclosed in the application form in future applications to avoid potential problems caused by missing information.
2 Actively monitor the effective use of DC and the number of storage providers and copies, SP good retrieval rate and data replication, SP geographical location.
In general, ask customers to provide more information, solve existing problems, and strengthen monitoring of DataCap allocation and use. Find feasible solutions through communication and coordination.

**Comment by filecoin-watchdog** [comment]
Comment regarding data stored by the only current client of this allocator:
https://github.com/Chuangshi1/Genesis/issues/44#issuecomment-2533374997

**Comment by Chuangshi1** [comment]
> Comment regarding data stored by the only current client of this allocator: [Chuangshi1/Genesis#44 (comment)](https://github.com/Chuangshi1/Genesis/issues/44#issuecomment-2533374997)

Yes, we notice the problem and ask the client to correct it now. And we will continuously monitor the entire process in the next days. Welcome to check at that time. At the same time，by learning from these experiences, we will be able to more effectively manage customer needs, optimize processes, and improve overall service quality in future projects.

**Comment by galen-mcandrew** [comment]
Overall seeing diligence and compliance issues here. It appears the allocator is responsive when issues are brought up, but there are large gaps in the allocator's diligence process and investigations. Some specific flags:

- data diligence and investigation
- accurate, updated, transparent KYC on clients
- dataset size calculations and allocation tranches
- data distribution and regional requirements

Overall, there is a troubling pattern happening here that I want to call out. A new client arrives, applies with incomplete information and minimal diligence, and then rapidly is awarded all of the DataCap that the allocator has. 
![Screenshot 2024-12-19 at 5 17 27 PM](https://github.com/user-attachments/assets/0c3918e9-863c-46e6-ab1f-b7e85c2870e7)

While the allocator is engaging with clients, and there is an increase in retrieval rates, the above issues need significant improvement during the next allocations. 

**Given this pattern and the flags mentioned above, we are requesting 2.5PiB of DataCap.**
